ansible/templates/usb/scripts/prompt-config.sh.j2

```diff
@@ -1347,6 +1362,18 @@ console=$(getanswer "config_console")
 # prompt:
 if [[ -z ${console} || $(tty) != "/dev/console" ]]; then
 	read continue;
+	# automatic installation is always advanced
+	printheader "Advanced installation"
+	printf "
```

Comment:
The messages are printed consistently with the following pattern throughout the script:

```
    message="
Blabla.\n\n"

    if [[ $(getanswer "skip_instructions") != "true" ]]; then
        printf "$message"
    fi
```
Please follow this pattern here.